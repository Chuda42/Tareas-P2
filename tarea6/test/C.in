hacerVecinos 1 2 1.0
sonVecinos 1 2
sonVecinos 9 1
distancia 1 2
vecinos 1
vecinos 2
vecinos 9
cantidadVertices
hacerVecinos 1 4 1.0
hacerVecinos 3 9 1.0
vecinos 1
vecinos 2
vecinos 4
cantidadVertices
insertarEnCP 4 4.0
insertarEnCP 9 9.0
insertarEnCP 3 3.0
existeAsociacionEnMap 5
prioridad 3
actualizarEnCP 9 30.0
eliminarPrioritario
prioritario
Fin
