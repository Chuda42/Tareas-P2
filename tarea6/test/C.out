1>Nuevos vecinos: 1 2 1.00.
2>Son vecinos.
3>NO son vecinos.
4>1.00
5>2 
6>1 
7>
8>10.
9>Nuevos vecinos: 1 4 1.00.
10>Nuevos vecinos: 3 9 1.00.
11>2 4 
12>1 
13>1 
14>10.
15>Se insert贸 el elemento.
16>Se insert贸 el elemento.
17>Se insert贸 el elemento.
18>No existe asociacion.
19>3.00.
20>Se actualiz贸 el valor.
21>Eliminado.
22>4
23>Fin.
